docs/conf.py

```diff
@@ -67,11 +67,22 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+extensions.append("sphinx_wagtail_theme")
```

Comment:
What's the difference between L70 compared to adding `"sphinx_wagtail_theme"` after L44?

Reply:
That's a good question. The quick start guide which I originally followed: https://sphinx-themes.org/sample-sites/sphinx-wagtail-theme/#documentation

But then the installation section for the docs says otherwise: https://sphinx-wagtail-theme.readthedocs.io/en/latest/usage.html

I'll revert adding it to the extensions list. Append shouldn't be needed for it.

Reply:
No worries. I was simply curious. 

BTW Sphinx has issues with finding the source directory unless you explicitly provide it which is why I think the RTD build is failing. 

I am struggling with a similar issue and I found a discussion on this helpful: https://stackoverflow.com/a/60159862/10241324

Reply:
I thought it was due to me not changing out the theme in `pyproject.toml`